controllers/admin/AdminCmsCategoriesController.php

```diff
@@ -258,36 +258,36 @@ public function renderForm()
                     'lang' => true,
                     'rows' => 5,
                     'cols' => 40,
-                    'hint' => $this->l('Invalid characters:').' &lt;&gt;;=#{}'
+                    'hint' => $this->trans('Invalid characters:', array(), 'Admin.Global').' &lt;&gt;;=#{}'
                 ),
                 array(
                     'type' => 'text',
-                    'label' => $this->l('Meta title'),
+                    'label' => $this->trans('Meta title', array(), 'Admin.Global'),
                     'name' => 'meta_title',
                     'lang' => true,
-                    'hint' => $this->l('Invalid characters:').' &lt;&gt;;=#{}'
+                    'hint' => $this->trans('Invalid characters:', array(), 'Admin.Global').' &lt;&gt;;=#{}'
                 ),
                 array(
                     'type' => 'text',
-                    'label' => $this->l('Meta description'),
+                    'label' => $this->trans('Meta description', array(), 'Admin.Parameters.Feature'),
                     'name' => 'meta_description',
                     'lang' => true,
-                    'hint' => $this->l('Invalid characters:').' &lt;&gt;;=#{}'
+                    'hint' => $this->trans('Invalid characters:', array(), 'Admin.Global').' &lt;&gt;;=#{}'
                 ),
                 array(
                     'type' => 'text',
-                    'label' => $this->l('Meta keywords'),
+                    'label' => $this->trans('Meta keywords', array(), 'Admin.Parameters.Feature'),
                     'name' => 'meta_keywords',
                     'lang' => true,
-                    'hint' => $this->l('Invalid characters:').' &lt;&gt;;=#{}'
+                    'hint' => $this->trans('Invalid characters:', array(), 'Admin.Global').' &lt;&gt;;=#{}'
                 ),
                 array(
                     'type' => 'text',
-                    'label' => $this->l('Friendly URL'),
+                    'label' => $this->trans('Friendly URL', array(), 'Admin.Parameters.Feature'),
                     'name' => 'link_rewrite',
                     'required' => true,
                     'lang' => true,
-                    'hint' => $this->l('Only letters and the minus (-) character are allowed.')
+                    'hint' => $this->trans('Only letters and the minus (-) character are allowed.', array(), 'Admin.Global')
```

Comment:
Nice. How about modules? How can I add Symfony translations from module folders? Or has it not been implemented, yet?